google/ggrc-core
# GGRC-6307 Fix layout on Person's Info panel

# Issue description

Layout of controls on Person's Info panel is broken.

# Steps to test the changes

1. Open any object page (e.g. Program)
2. Click on Peoples tab
3. Open click on any person in tree view
Actual Result: Popup layout is broken
Expected Result: Layout should not be broken: 3 dots menu, close, max/min should be displayed on right top corner of Info panel. There should also be person name, email, company name, authorizations and custom attributes.

# Solution description

There are two templates with person info: **people/info.mustache** (is used on the /people/**id**#!info page) and **people_roles/info.mustache** (is used for info-pin). The layout of popup should be the same as on the Person page, so there is no need to have the separate `people_roles/info.mustache` file and it was removed. Layout in the `people/info.mustache` has also been fixed and improved.

# Sanity checklist




# PR Review checklist
